.github/workflows/jira.yaml

```diff
@@ -53,6 +53,24 @@ jobs:
           echo "type=ISS" >> "$GITHUB_OUTPUT"
         fi
 
+        curl -o repo-teams.txt https://raw.githubusercontent.com/hashicorp/vault-workflows-common/main/repo-teams.txt
+
+        echo "::debug::Determining team labels"
+        org_with_repo=${{ github.event.repository.name }}
+        repo="${org_with_repo#hashicorp/}"
+        label="$(grep --max-count 1 $repo repo-teams.txt | awk '{print $2}')"
+        # append team label to array
+        teams_field='[]'
+        teams_field="$(echo "$teams_field" | jq --compact-output --arg tt "$label" '. |= . + [$tt]')"
```

Comment:
This jq nonsense is probably not necessary, I just copied it from our previous override logic that used a for loop. Happy to change it if anyone feels strongly.

Reply:
This bugged me so I changed it.